README.md

```diff
@@ -68,6 +68,15 @@ If there are changes made in the Original repository, you can sync the fork to k
 3. ```git checkout master``` to checkout your fork's local master branch.
 4. ```git merge upstream/master``` to sync your local master' branch with upstream/master. Note: Your local changes will be retained and your fork's master branch will be in sync with the upstream repository.
 
+#### Configure the app
+
+Maps app uses authenticated ArcGIS Online services for routing and geocoding. You need to configure the app for OAuth authentication.
+
+1. Log in to developers.arcgis.com.
+2. Create a [new application](https://developers.arcgis.com/applications/new).
+3. After creating the application, navigate to the 'Authentication' tab and add a redirect URI. Note the Client ID and Redirect URI.
```

Comment:
> Note the Client ID and Redirect URI.

At this step, there will not yet be a specified Redirect URI.  A few questions here, does the maps app dot net require a Redirect URI? (The iOS counterpart so I don't know why it wouldn't) If so, is it configurable? And can we improve the README setup instructions to ensure a user knows how to configure the app in this aspect?
```suggestion
3. After creating the application, navigate to the 'Authentication' tab and add a redirect URI. Note the **Client ID** and **Redirect URI**.
```